docs/source/installation/install_with_package_manager.rst

```diff
@@ -1,7 +1,79 @@
 Install immuneML with a package manager
 =========================================
 
-Steps to install immuneML with Anaconda (tested with version 4.8.3):
+This manual shows how to install immuneML with `Anaconda <https://docs.anaconda.com/anaconda/install/>`_ (tested with version 4.8.3).
+
+While the immuneML core functionalities do not depend on R, R dependencies are necessary to generate certain plots (see: :ref:`When should I install immuneML with R dependencies?`).
+Installing immuneML with R dependencies takes a few more steps. Therefore, both tutorials are shown separately below.
+
+
+Install immuneML without R dependencies
+---------------------------------------
+
+1. Create a directory for immuneML and navigate to the directory:
+
+.. code-block:: console
+
+  mkdir immuneML/
+  cd immuneML/
+
+2. Create a virtual environment using conda:
+
+.. code-block:: console
+
+  conda create --prefix immuneml_env/ python=3.7.6
+
+3. Activate the created environment:
+
+.. code-block:: console
+
+  conda activate immuneml_env/
+
+4. Install basic immuneML including Python dependencies from GitHub using pip:
+
+.. code-block:: console
+
+  pip install git+https://github.com/uio-bmi/immuneML
+
+Alternatively, if you want to install immuneML including optional extras (:code:`DeepRC`, :code:`TCRDist`), use:
+
+.. code-block:: console
+
+  pip install git+https://github.com/uio-bmi/immuneML#egg=immuneML[DeepRC,TCRDist]
+
+Installing DeepRC and TCRDist dependencies is necessary to use the :ref:`DeepRC` and :ref:`TCRDISTClassifier` ML methods.
```

Comment:
it is also necessary for TCRdistMotifDiscovery report.

Reply:
updated :)